shelley/chain-and-ledger/formal-spec/protocol-parameters.tex

```diff
@@ -114,9 +123,19 @@ \section{Protocol Parameters}
     \end{array}
   \end{equation*}
   %
-  \emph{Derived Functions}
+  \caption{Definitions used in Protocol Parameters}
+  \label{fig:defs:protocol-parameters}
+\end{figure*}
+
+\begin{figure*}[htb]
+  \emph{Helper Functions}
   %
   \begin{align*}
+    \fun{minfee} & \in \PParams \to \Tx \to \mathbb{Z} & \text{minimum fee}\\
+    \fun{minfee} & ~\var{pps}~\var{tx} =
```

Comment:
```minfee``` is defined above as an abstract function with a different type, evaluating to `Coin`

Reply:
good catch, I removed the abstract one